packages/app/src/elements/audit.ts

```diff
@@ -0,0 +1,214 @@
+
+    static styles = [shared, css``];
+
+    render() {
+        const { listItems, label, routeParams } = this;
+
+        return html`
+            <section class="box count" ?hidden=${listItems.length === 0}>
+                <h2 class="uppercase bg-dark border-bottom semibold center-aligning spacing horizontal layout">
+                    <div></div>
+                    <div>${label}</div>
+                    <div class="tiny bold tag">${listItems.length}</div>
+                    <div class="stretch"></div>
+                </h2>
+                <pl-list>
+                    ${listItems.slice(0, 5).map(
+                        (listItem) => html`
+                            <div class="list-item hover click" @click=${() => this.go(`items/${listItem.item.id}`)}>
+                                <pl-vault-item-list-item
+                                    .item=${listItem.item}
+                                    .vault=${listItem.vault}
+                                    class="stretch collapse"
+                                ></pl-vault-item-list-item>
+                            </div>
+                        `
+                    )}
+                </pl-list>
+                <pl-button
+                    class="slim margined transparent"
+                    @click=${() => this.go("items", routeParams)}
+                    ?hidden=${listItems.length < 6}
+                >
+                    <div>${$l("Show All")}</div>
+                    <pl-icon icon="arrow-right" class="small left-margined"></pl-icon>
+                </pl-button>
+            </section>
+        `;
+    }
+}
+
+@customElement("pl-audit")
+export class Audit extends StateMixin(LitElement) {
+    @state()
+    private _reusedPasswords: ListItem[] = [];
+
+    @state()
+    private _weakPasswords: ListItem[] = [];
+
+    @state()
+    private _compromisedPasswords: ListItem[] = [];
+
+    @queryAll("pl-audit-list-item")
+    private _countElements: HTMLDivElement[];
+
+    static styles = [
+        shared,
+        css`
+            :host {
+                display: block;
+            }
+
+            .counts {
+                display: grid;
+                grid-gap: 1em;
+                margin: 1em;
+                grid-template-columns: repeat(auto-fit, minmax(20em, 1fr));
+            }
+
+            @media (max-width: 700px) {
+                .counts {
+                    grid-template-columns: repeat(auto-fit, minmax(15em, 1fr));
+                }
+            }
+        `,
+    ];
+
+    render() {
+        const { _reusedPasswords, _weakPasswords, _compromisedPasswords } = this;
+
+        // TODO: Remove this
+        console.log("======== rendering!");
+        console.log(
+            JSON.stringify({
+                reusedPasswordsCount: _reusedPasswords.length,
+                weakPasswordsCount: _weakPasswords.length,
+                compromisedPasswordsCount: _compromisedPasswords.length,
+            })
+        );
```

Comment:
@MaKleSoft I'm sure this is a noob-in-lit kind of problem, but I can get new values here, while the rendered component only shows whatever came the first time around. I even tried `this.stateChanged();` in `this.audit()` to force it to update, but no go. I also played around with `@property` and `@state` inside the `pl-audit-list-item`, in case that was the thing not updating.

What am I doing wrong?